.editorconfig

```diff
@@ -0,0 +1,17 @@
+root = true
+
+
+[*]
+
+# Change these settings to your own preference
+indent_style = space
```

Comment:
I'm more a tabs person :P but this is a common .editorconfig in auth0 codebase and I copy paste it to any new repo. I prefer having it submitted so that any contributor follows the same indentation and we don't end up with a mix of tabs and spaces or different indent sizes

Reply:
Ahhh makes sense. :+1: